arangodb-net-standard.Test/CollectionApi/CollectionApiClientTest.cs

```diff
@@ -261,5 +261,63 @@ public async Task GetCollectionPropertiesAsync_ShouldSucceed()
             Assert.Equal(3, response.Status);
             Assert.Equal(2, response.Type);
         }
+        
+        [Fact]
+        public async Task RenameCollectionAsync_ShouldSucceed()
+        {
+            await _adb.Collection.PostCollectionAsync(
+                    new PostCollectionBody
+                    {
+                        Name = "TempCollection"
+                    });
+            var response = await _collectionApi.RenameCollectionAsync("TempCollection", new RenameCollectionRequest
+            {
+                Name = "testingCollection"
+            });
+            Assert.Equal(HttpStatusCode.OK, response.Code);
+            Assert.Equal("testingCollection", response.Name);
+            Assert.False(response.IsSystem);
+            Assert.NotNull(response.Id);
+            Assert.False(response.Error);
+        }
+
+        [Fact]
+        public async Task RenameCollectionAsync_ShouldThrow_WhenCollectionNotFound()
+        {
+            var exception = await Assert.ThrowsAsync<ApiErrorException>(async () =>
+            {
+                await _collectionApi.RenameCollectionAsync("bogusCollection", new RenameCollectionRequest
+                {
+                    Name = "testingCollection"
+                });
+            });
+            Assert.Equal(HttpStatusCode.NotFound, exception.ApiError.Code);
```

Comment:
Here we only test for `Code`, but in the other test methods below we only test for `ErrorEnum`. Should we test both in all cases? The idea is to make sure our data model carries the properties correctly.

Reply:
Yes we should. Cause I have noticed that they do differ. Like when I developed the GetGraphAsync and tested for not found. The COde was standard 404 but the ErrorNum was 1924 for Graph not found. So I'll add a test for error num aswell